### DISCUSSION TITLE
MLB Home Run Race in the 1998 season

### DISCUSSION THREAD

**Discussion by StuffbyYuki** [has2k1/plotnine#820] [2024 Plotnine Contest]
### Authors

Yuki Kakegawa 

### Links

[linkedin](https://www.linkedin.com/in/yukikakegawa/), [consulting company](https://www.oremdata.com/), [personal website](https://yukikakegawa.me/), [medium](https://medium.com/@yukithejapanese)

### Full description

Recreation of the viz found in [Chapter 3: Graphics - Analyzing Baseball Data with R (3e)](https://beanumber.github.io/abdwr3e/03-graphics2.html), displaying the cumulative sum of home runs among a few top players in the 1998 season.

### Code repository

https://github.com/StuffbyYuki/Plotnine-Contest/blob/main/2024/analyzing_baseball_data_with_r/1998_homerun_race.ipynb

**Comment by has2k1** [comment]
![hr1](https://github.com/user-attachments/assets/f97c32b5-26e6-4bd1-bf0c-b1f1afcc29bf)
<hr/>

![hr2](https://github.com/user-attachments/assets/3aa421f5-c624-49d7-9d20-993ad1dfaa8e)

**Comment by has2k1** [comment]
@StuffbyYuki, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
